website/docs/reference/resource-configs/trino-configs.md

```diff
@@ -365,3 +365,7 @@ models:
 ```
 </File>
 
+## Model contracts
+
+The `dbt-trino` adapter supports [model contracts](/docs/collaborate/govern/model-contracts). Currently, only [constraints](/reference/resource-properties/constraints) with `type` as `not_null` are supported.
+Before using `not_null` constraint in your model, make sure the underlying connector supports `not null`.
```

Comment:
you're right, @damian3031 , that would be too much. how about this or something similar? 

```suggestion
Before using `not_null` constraints in your model, make sure the underlying connector supports `not null` to avoid running into errors.```